### PR TITLE
Add static-libs installation

### PR DESCRIPTION
**Pull request by driverxdw**
add static library installation in CMakeLists.txt of libclamav submodules

### CODE REVIEW

**Comment by micahsnyder** [comment]
Manual testing on Linux looks good:
```
❯ ls -la install/lib/
total 42252
drwxr-xr-x 3 micah micah     4096 Nov 27 10:36 ./
drwxr-xr-x 8 micah micah     4096 Nov 27 10:36 ../
-rw-r--r-- 1 micah micah 38642020 Nov 27 10:36 libclamav_rust.a
-rw-r--r-- 1 micah micah  3696484 Nov 27 10:36 libclamav_static.a
-rw-r--r-- 1 micah micah   145808 Nov 27 10:35 libclammspack_static.a
-rw-r--r-- 1 micah micah    16110 Nov 27 10:35 libclamunrar_iface_static.a
-rw-r--r-- 1 micah micah   633086 Nov 27 10:35 libclamunrar_static.a
-rw-r--r-- 1 micah micah   108684 Nov 27 10:36 libfreshclam_static.a
drwxr-xr-x 2 micah micah     4096 Nov 27 10:36 pkgconfig/
```

File: libclamunrar_iface/CMakeLists.txt [hunk 1]
```diff
@@ -109,6 +109,13 @@ if(ENABLE_UNRAR)
             PRIVATE
                 ClamAV::libunrar_static )
 
+        if(WIN32)
+            install(TARGETS clamunrar_iface_static DESTINATION . COMPONENT libraries)
+            install( FILES $<TARGET_PDB_FILE:clamunrar_iface> DESTINATION . OPTIONAL COMPONENT libraries )
```
Comment (micahsnyder) [inline libclamunrar_iface/CMakeLists.txt:114]:
```suggestion
```
This line is not required. The target name is wrong, but also PDB files do not exist for static libraries.

**Review by micahsnyder — CHANGES_REQUESTED** [review]
I found that the windows static build fails without these suggested changes.

**Review by driverxdw — COMMENTED** [review]
I've fixed it, thank you for your suggestion!

**Review by micahsnyder — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**